### ISSUE TITLE
Functionality to show missed Incubated dates?

### ISSUE THREAD

**Issue by lanza** [Trevoke/org-gtd.el#186]
e.g. with

```
* Incubate
** Buy house
:PROPERTIES:
:ID:       Buy-house-2023-07-10-01-51-15
:ORG_GTD_TIMESTAMP: <2023-07-10>
:END:
<2023-07-10>
```

How can one go about finding that this is missed without manually auditing the todo file? `org-gtd-oops` doesn't look for this:

https://github.com/Trevoke/org-gtd.el/blob/master/org-gtd-oops.el#L43

and `org-gtd-review-stuck-incubated-items` is only looking for date-less entries:

https://github.com/Trevoke/org-gtd.el/blob/master/org-gtd-review.el#L133

Previously, I've just used `org-agenda-later` to find this information, but that seems to break the custom agenda configuration that `org-gtd` sets up and `org-agenda-later` followed immediately by `org-agenda-earlier` shows a blanked out version of the `org-gtd-engage` agenda:
Before:
<img width="293" alt="image" src="https://github.com/Trevoke/org-gtd.el/assets/12656790/7a0274b0-b5f3-40c0-9a20-e6265c2d8aff">
After:
<img width="294" alt="image" src="https://github.com/Trevoke/org-gtd.el/assets/12656790/669905eb-48e4-4310-8128-b2698b2a8572">

**Comment by Trevoke** [comment]
You're totally right, that's completely missing. I thought there was an open feature request for this already but clearly not, so thanks for opening it, and... There's a PR to look at <3

**Comment by Trevoke** [comment]
Wow this issue should be closed, the PR got merged in forever ago
